Source/Request.swift

```diff
@@ -308,7 +308,12 @@ extension Request: CustomDebugStringConvertible {
                 let cookies = cookieStorage.cookies(for: url), !cookies.isEmpty
             {
                 let string = cookies.reduce("") { $0 + "\($1.name)=\($1.value);" }
-                components.append("-b \"\(string.substring(to: string.characters.index(before: string.endIndex)))\"")
+
```

Comment:
No need for extra whitespace around the `#if` checks. We prefer to keep the whitespace the same as if there was no check in the first place.